### DISCUSSION TITLE
Flat Timer

### DISCUSSION THREAD

**Discussion by 8BitAvenger** [Senryoku/Draftmancer#646] [Ideas]
Is there any way currently or could there be a way added to use a more customized timer?

I have an application where the users would prefer an 18-hour timer or 24-hour timer for each individual pick, and though the default of lowering the turn timer per pick makes obvious sense in just about every live drafting situation, it'd be great to have our picks start at 24 hour and have the 14th pick be 18 hours, or just flat 24 or flat 18.

**Comment by Senryoku** [comment]
Mmh, I could add more customization, but the timer system wasn't really designed for that: It will pause if any player disconnects. Do you intent to have all players continuously connected for days?
Could you describe your use case in more details?

**Reply by 8BitAvenger** [reply]
Ah, I forgot about the auto-pause-on-disconnect. The use case is asynchronous daily cube draft picking then we play Round Robin bo3 matches amongst ourselves super lax and chill over the course of the next couple weeks after the slow draft is over. We have had a Discord bot for the draft portion that's currently on the fritz and not being maintained.

I think asking you to make the changes to help us considering we'd be .0001% of your active userbase would be quite silly, especially considering the auto-pause-on-disconnect, which would clearly need to be disabled for our use case. Thanks for the reply and for your time though! Love the site and will continue to use the heck out of it for live-drafting. =D

**Reply by Senryoku** [reply]
Yeah, I sometimes wonder if it *should* pause, but it's one of these things that would require a significant rewrite for very little gain (if any for most users)

Maybe checkout Rotisserie Draft! I mean... It's a different format and doesn't even have a timer, but it was designed to be ran asynchronously over the course of multiple days ^^
